src/e2e-node/.env.defaults

```diff
@@ -1,12 +1,19 @@
-# Save a copy of this file as .env.local to prevent your credentials from being committed
+# Save a copy of this file as .env.local to prevent your credentials from being
+# committed.
+# These credentials determine what Pod the end-to-end tests will write data to.
+# Generally, the tests should clean up after themselves, but it is advisable to
+# use a Pod created specifically for running the end-to-end tests.
+#
+# To obtain a refresh token, client ID and client secret for your account, run
+#     npx @inrupt/generate-oidc-token
```

Comment:
I discussed mentioning this explicitly in here with NicAS already, and we agreed not to. Could add a reference back to README instead (i.e., "For guidance on how to generate these values, see the README.md...") - just so the information is only maintained in one place instead of multiple.

Reply:
Good point. I merged in your README and removed this from `.env.defaults`: 7d5bcfd6

Reply:
Ah and 10bab4a4 of course.